tracing/src/macros.rs

```diff
@@ -2017,6 +2063,21 @@ macro_rules! fieldset {
         $crate::fieldset!(@ { $($out),*, $crate::__tracing_stringify!($($k).+) } $($rest)*)
     };
 
+    (@ { $(,)* $($out:expr),* } $k:literal = ?$val:expr, $($rest:tt)*) => {
```

Comment:
Again, maybe worth adding a comment to separate the two blocks of cases.

Reply:
done